recipe/gen_patch_json.py

```diff
@@ -2548,6 +2548,26 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
             new_constrains.append("jpeg <0.0.0a")
             record["constrains"] = new_constrains
 
+        # cppyy <3 uses a version of Cling that is based on Clang 9. libcxx 15
+        # headers for macOS do not compile with such an old Clang anymore, see
+        # https://github.com/conda-forge/libcxx-feedstock/issues/111
+        # So, if there's is no "<" pin on libcxx already, we add a "<15".
+        if (
+            record_name == "cppyy" and
+            pkg_resources.parse_version(record["version"]) < pkg_resources.parse_version("3.0.0")
```

Comment:
```suggestion
            pkg_resources.parse_version(record["version"]) < pkg_resources.parse_version("3.0.0") and
            record.get("timestamp", 0) < OUTPUT_OF_PROVIDED_COMMAND
```

Reply:
```
python -c "import time; print(f'{time.time():.0f}000')"
```
^^^ Run this command from your terminal, and paste the output

Reply:
Thanks I'll open a separate PR to improve the checklist. (Maybe, I'm the only one who doesn't get it though…)